src/modules/workspace/production.ts

```diff
@@ -21,7 +21,7 @@ const pretty = p => p ? chalk.green('production mode') : chalk.red('development
 export default async (production: any) => {
   let prod
 
-  if (production === null || production === 'true') {
+  if (production === null || production === 'true' || production === undefined) {
```

Comment:
The only reason I will not accept your suggestion is that it would do the opposite of what we expect in case `production` is the boolean value `false`. Other than that, it is a good suggestion. 🙂